Documentation/ceph-nfs-crd.md

```diff
@@ -4,19 +4,30 @@ weight: 3100
 indent: true
 ---
 
-# Ceph NFS Gateway CRD
+# Ceph NFS Server CRD
 
 ## Overview
```

Comment:
Overall, I'm wondering if we could write this entire doc assuming v16.2.7 is the minimum version. Documenting all the version combinations will be difficult to maintain so would be good to avoid it unless there is an explicit need. A section on upgrading from previous versions is certainly needed though, but if we can only discuss previous versions in the context of upgrades, I would vote for that.

Reply:
Octopus isn't likely to change, and I already wrote the docs up for it. It will likely be helpful to some users, and I don't see the harm in keeping it.

Reply:
Ok, how about rearranging a bit so the v16 topics are mentioned first? Mainly, my goal for the readers is that they can easily glance at which part of the doc they can focus on and quickly skip the version that they're not using. Hopefully most are already on v16 at this point, thus to cover it first.

Reply:
Sounds good, and done.